tfs/pom.xml

```diff
@@ -11,6 +11,12 @@
   <name>Team Foundation Server Plug-in</name>
   <version>5.126.0-SNAPSHOT</version>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Team+Foundation+Server+Plugin</url>
+
+  <properties>
+    <!-- TODO: enable once FindBugs issues are fixed -->
+    <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
Is this necessary? I hesitate to turn it off.

Reply:
With the current configuration you will be getting FindBugs warnings in the build log. They just won't fail the build.